10-inference.Rmd

```diff
@@ -256,42 +244,43 @@ population_parameters
 ```
 The price per night of all Airbnb rentals in Vancouver, BC is \$`r round(population_parameters$pop_mean,2)`, on average. This value is our population parameter since we are calculating it using the population data. 
```

Comment:
probably should bold important terms (population parameter) again to mimic the previous section. seems like we do that below with point estimate already